content/doc/tutorials/using-jenkinsfile-runner-github-action-to-build-jenkins-pipeline.adoc

```diff
@@ -0,0 +1,378 @@
+section: doc
+---
+
+:toc:
+:toclevels: 3
+:imagesdir: ../../book/resources
+
+This tutorial shows you how to use link:https://jenkinsci.github.io/jfr-action-doc/[Jenkinsfile Runner GitHub Actions] in the link:https://github.com/features/actions[GitHub Actions] context. 
+
+=== The concept
+
+Classical Jenkins instances bind physically with the permanently running servers. 
+link:https://github.com/jenkinsci/jenkinsfile-runner[Jenkinsfile Runner] packages the Jenkins core and other necessary items, and serves as an entry point to your pipeline job.
+At a high level, Jenkinsfile Runner GitHub Actions wrap around the Jenkinsfile Runner and other necessary modules for the user. 
+Once the user commits the changes to the remote GitHub repository, the GitHub Actions will run the pipeline as defined by the Jenkinsfile. 
+After the pipeline ends, the user can view the pipeline log in the GitHub Action page. 
+Furthermore, the user can integrate the GitHub Actions in the marketplace with the Jenkinsfile Runner GitHub Actions.
+
+=== Prerequisites
+
+For this tutorial, you will need:
+
+* A GitHub account
+* A GitHub repository
+* Git installed locally
+* Fundamental knowledge link:https://docs.github.com/en/actions[of GitHub Actions and how they work].
+* If you want to integrate Jenkins plugins, you also need to know what link:https://www.jenkins.io/projects/jcasc/[JCasC] is. 
+This tutorial reviews some basic concepts of JCasC.
+
+=== Create a GitHub repository
+
+We recommend users refer to link:https://docs.github.com/en/get-started/quickstart/create-a-repo[the GitHub official guide] as a starter.
+
+* In the upper-right corner of GitHub, use the “+” drop-down menu, then select *New repository*.
+* Provide a memorable name for your repository.
+For example, "hello-world".
+* Optionally, add a description of your repository.
+For example, "My first repository on GitHub".
+* Choose a repository visibility.
+
+[NOTE]
+====
+GitHub Actions are free for standard GitHub-hosted runners in public repositories and self-hosted runners. 
+If you make the repository private, each GitHub account receives a predetermined amount of free minutes and storage for use with GitHub-hosted runners. 
+Please refer to link:https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions[the billing and pricing of GitHub Actions] for more information.
+====
+
+=== Create a Jenkinsfile
+
+In this section you will create a Jenkinsfile, which defines how to run the Jenkins pipeline. 
+Refer to the link:https://docs.github.com/en/actions[official manual] for more details about the workflow definitions. 
+The following is an example Jenkinsfile, where the name of this Jenkinsfile is “Jenkinsfile”:
+[source,groovy]
+----
+pipeline {
+    agent any
+    stages {
+        stage('hello') {
+            steps {
+                sh 'echo Hello Jenkins!'
+            }
+        }
+    }
+}
+----
+
+=== Create a plugin list file
```

Comment:
I see. Because the simplest example (the sections without **optional** tag) given in this tutorial doesn't require to install extra plugins, I didn't mention how to describe the plugin by the plugin list.

Let me update it later.

Reply:
Already add it in the latest commit.